m2cgen/interpreters/r/interpreter.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+from m2cgen.interpreters import mixins, utils
+from m2cgen.interpreters.interpreter import ToCodeInterpreter
+from m2cgen.interpreters.r.code_generator import RCodeGenerator
+
+
+class RInterpreter(ToCodeInterpreter,
+                   mixins.LinearAlgebraMixin,
```

Comment:
I believe this mixing is no longer needed, isn't it?

Reply:
It's needed to set `self.with_linear_algebra = True`. Just like it's done in Python interpreter: https://github.com/BayesWitnesses/m2cgen/blob/9072ba010bbabfc8f9c9ad8bf9747f0c8a5d2af2/m2cgen/interpreters/python/interpreter.py#L40

Reply:
Ah, overlooked that. Sounds good 👍